django/utils/version.py

```diff
@@ -77,3 +78,14 @@ def get_git_changeset():
     except ValueError:
         return None
     return timestamp.strftime('%Y%m%d%H%M%S')
+
+
+def get_version_tuple(version):
+    """Return tuple version (X, Y, Z) from VERSION if present."""
```

Comment:
Why is VERSION capitalized? Maybe something like:

Return a tuple of version numbers (e.g. (1, 2, 3)) from the version string (e.g. '1.2.3').